.github/docker/simple-server.compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
```

Comment:
Can this is maintained on the server repo or does it have to be here? I want to avoid potential drift of setup in future between server docker file changes and this.

During CI we can checkout the server repo and use that file as well.